rpm/asteroidsyncservice.spec

```diff
@@ -40,8 +40,9 @@ Support for AsteroidOS watches in SailfishOS.
 rm -rf %{buildroot}
 %qmake5_install
 
-mkdir -p %{buildroot}%{_libdir}/systemd/user/user-session.target.wants
-ln -s ../asteroidsyncserviced.service %{buildroot}%{_libdir}/systemd/user/user-session.target.wants/
+echo %{_userunitdir}
```

Comment:
Did you mean to keep this echo? :) I think it's ok for debugging but not worth keeping in the spec

Reply:
Yeah that's a debug line for sure :)